.storybook/main.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+
+const language = process.env.LANGUAGE;
+
+const config: { webpackFinal: (config: any) => Promise<any>; stories: string[]; framework: { name: string }; addons: (string | { name: string; options: { sass: { implementation: any } } })[] } = {
```

Comment:
The type that was provided in the documentation for this situation was not working in TS, so I let intellij generate one